### DISCUSSION TITLE
Connection costs not properly factored in or discounted

### DISCUSSION THREAD

**Discussion by hamishbeath** [CLOVER-energy/CLOVER#195] [IAA]
### Describe the bug
When specifying connection costs, the model should calculate the connection costs for initial households as well as new ones. Currently the model misses initial connection costs due to the way the connections_expenditure function is written in the finance.py file. Further, it is more accurate to discount new connections to the year they are connected rather than the start of the increment year. 

### To Reproduce
1. Run a simulation with a positive connection cost and number of households 
2. See the incorrect amounts in the appraisal outputs (when compared to calculating on excel) 

### Expected behaviour
The model should factor in initial connection costs e.g. 100 initial households * connection cost

**Comment by BenWinchester** [comment]
## General thoughts
This is definitely a bug within CLOVER, but I'm not sure that it should belong under the "IAA" header? @hamishbeath , we could address this as part of CLOVER 6.0 perhaps but leave it when considering the IAA stream? 😃
